code/go/0chain.net/blobbercore/challenge/protocol.go

```diff
@@ -301,6 +304,11 @@ func (cr *ChallengeEntity) LoadValidationTickets(ctx context.Context) error {
 }
 
 func (cr *ChallengeEntity) CommitChallenge(ctx context.Context, verifyOnly bool) error {
+	if time.Since(cr.CreatedAt) > config.Configuration.ChallengeCompletionTime+time.Second {
```

Comment:
@cnlangzi @lpoli we should not set cct in config
but get from this API `sharder01/v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/storage-config` key: `max_challenge_completion_time`

Reply:
Lets make another PR.
For cct, blobber should call sharder once and store this value in some unexported variable exposed with exported function